src/components/com_tjcertificate/administrator/views/trainingrecord/tmpl/edit.php

```diff
@@ -57,7 +57,7 @@
 		<?php
 			echo HTMLHelper::_('bootstrap.startTabSet', 'myTab', array('active' => 'general'));
 			echo HTMLHelper::_('bootstrap.addTab', 'myTab', 'general', Text::_('COM_TJCERTIFICATE_TITLE_CERTIFICATE')); ?>
-		<div class="row-fluid">
+		<div class="row ptb-20 plr-10">
```

Comment:
@MeghaBiranje row-fluid is for bs2 and this might be needed for joomla 3
create separate layouts for bs3 and bs5 if needed